data/data/baremetal/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```

Comment:
We should discuss how to get here. but not right now.

Reply:
Does this mean you want all OWNERS files removed?

Reply:
convert approvers to reviewers in the OWNERS files.